frontend/src/metabase/admin/upsells/UpsellUploads.tsx

```diff
@@ -0,0 +1,33 @@
+import { jt, t } from "ttag";
+
+import { getPlan } from "metabase/common/utils/plan";
+import { useSelector } from "metabase/lib/redux";
+import { getSetting } from "metabase/selectors/settings";
+
+import { UpsellCard } from "./components";
+
+export const UpsellUploads = ({ source }: { source: string }) => {
+  const plan = useSelector(state =>
+    getPlan(getSetting(state, "token-features")),
+  );
+
+  const showUpsell = plan === "oss" || plan === "starter";
+
+  if (!showUpsell) {
+    return null;
+  }
+
+  return (
+    <UpsellCard
+      title={t`Manage your uploads`}
+      campaign="manage-uploads"
+      buttonText={t`Try for free`}
+      buttonLink="https://www.metabase.com/upgrade"
```

Comment:
@sloansparger I didn't forget about this

[ENG-14599](https://linear.app/metabase-inc/issue/ENG-14599/export-commonly-used-upsell-links-from-a-single-file)